### PR TITLE
Api

### PR DESCRIPTION
**Pull request by JuanDavidBuitrago**
**Merge checklist:**
Check off if complete *or* not applicable:
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Fixup commits are squashed away
- [ ] Unit tests added/updated
- [ ] Manual testing instructions provided
- [ ] Noted any: Concerns, dependencies, migration issues, deadlines, tickets

### CODE REVIEW

File: platform_global_teacher_campus/models.py [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=100)
+    parentCategory = models.IntegerField()
+
+    def __str__(self):
+        return self.name
+
+class ValidationProcess(models.Model):
+    id = models.AutoField(primary_key=True)
+    courseId = models.ForeignKey(CourseOverview, on_delete=models.CASCADE)
```
Comment (Alec4r) [inline platform_global_teacher_campus/models.py]: This shouldn't be deleted even CourseOverview is delete, please change models.CASCADE to SET_NULL.

File: platform_global_teacher_campus/models.py [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+    currentValidationUserId = models.IntegerField()
+    validationBodyId = models.IntegerField()
+
+    def __str__(self):
+        return f"Validation Process for Course {self.courseid}"
+
+class ValidationBody(models.Model):
+    id = models.AutoField(primary_key=True)
+    validator = models.CharField(max_length=50)
+    name = models.CharField(max_length=100)
```
Comment (Alec4r) [inline platform_global_teacher_campus/models.py:85]: What is the reason for max_length 100?

File: platform_global_teacher_campus/models.py [hunk 3]
```diff
@@ -1,3 +1,33 @@
 """
 Database models for platform_global_teacher_campus.
 """
+
+from django.db import models
+from edxapp_wrapper.courses import get_course_overview
+
+CourseOverview=get_course_overview()
+class CourseCategory(models.Model):
+    id = models.AutoField(primary_key=True)
```
Comment (Alec4r) [inline platform_global_teacher_campus/models.py]: This is added by default by django, isn't it?

File: platform_global_teacher_campus/models.py [hunk 4]
```diff
@@ -1,3 +1,33 @@
...
+    def __str__(self):
+        return self.name
+
+class ValidationProcess(models.Model):
+    id = models.AutoField(primary_key=True)
+    courseId = models.ForeignKey(CourseOverview, on_delete=models.CASCADE)
+    categories = models.ManyToManyField(CourseCategory)
+    organizationId = models.IntegerField()
+    currentValidationUserId = models.IntegerField()
+    validationBodyId = models.IntegerField()
```
Comment (Alec4r) [inline platform_global_teacher_campus/models.py]: Why integer fields? why these are not a relations with each model?

File: platform_global_teacher_campus/models.py [hunk 5]
```diff
@@ -1,3 +1,33 @@
...
+
+    def __str__(self):
+        return f"Validation Process for Course {self.courseid}"
+
+class ValidationBody(models.Model):
+    id = models.AutoField(primary_key=True)
+    validator = models.CharField(max_length=50)
+    name = models.CharField(max_length=100)
+    admin_notes = models.CharField(max_length=100)
+    organizations = models.CharField(max_length=50)
```
Comment (Alec4r) [inline platform_global_teacher_campus/models.py]: Why CharField?? this should be a relation of many to many with organizations model.

File: platform_global_teacher_campus/models.py (hunk 1 above)
Comment (Asespinel) [reply platform_global_teacher_campus/models.py]: Ok I'll change it accordingly

File: platform_global_teacher_campus/models.py (hunk 2 above)
Comment (Asespinel) [reply platform_global_teacher_campus/models.py:85]: It was just a thought to limit the name to reduce the db's size but we can increase or delete the limit if needed

File: platform_global_teacher_campus/models.py (hunk 4 above)
Comment (Asespinel) [reply platform_global_teacher_campus/models.py]: Yes, you're right these relationships were temporarily defined until the other classes were written

File: platform_global_teacher_campus/models.py [hunk 6]
```diff
@@ -1,3 +1,56 @@
...
+    def __str__(self):
+        return self.name
+
+
+class ValidationProcess(models.Model):
+    courseId = models.OneToOneField(CourseOverview, on_delete=models.SET_NULL)
+    categories = models.ManyToManyField(CourseCategory)
+    organizationId = models.IntegerField()
+    currentValidationUserId = models.ForeignKey('Users', on_delete=models.SET_NULL, null=True)
+    validationBodyId = models.ForeignKey(ValidationBody, on_delete=models.SET_NULL, null=True)
```
Comment (Alec4r) [inline platform_global_teacher_campus/models.py]: - please remove Id
- Names validation_body, current_validation_body

follow good practices: https://steelkiwi.com/blog/best-practices-working-django-models-python/.

**Pull request merged**